packages/lisk-transactions/src/transactions/helpers/verify_multisignatures.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { TransactionError } from '../../errors';
+import { TransactionJSON } from '../../transaction_types';
+import { verifySignature } from './verify_signature';
+
+interface VerifyReturn {
+	readonly verified: boolean;
+	readonly errors: ReadonlyArray<TransactionError>;
+}
+
+export const verifyMultisignatures = (
+	memberPublicKeys: ReadonlyArray<string> = [],
+	minimumValidations: number = 0,
+	transaction: TransactionJSON,
+): VerifyReturn => {
+	if (
+		!transaction.signatures ||
+		transaction.signatures.length < minimumValidations
+	) {
+		return {
+			verified: false,
+			errors: [
+				new TransactionError(
+					'Missing signatures',
+					transaction.id,
+					'.signatures',
+				),
+			],
+		};
+	}
+
+	// tslint:disable-next-line no-unnecessary-type-assertion
```

Comment:
why do we need this?

Reply:
`transaction.signatures.forEach` below throws tslint `possible undefined`, so casting in assignment of `transactionSignatures`. however this is turn shows `no-unnecessary-type-assertion` lint error so disabling that here.

Reply:
In this case, definition of signatures should not be optional or should it be checked if signatures is undefined?

Reply:
mmm strange ='(